index.js

```diff
@@ -8,6 +8,17 @@ const normalizeOptions = require("./src/options").normalize;
 const parser = require("./src/parser");
 const printDocToDebug = require("./src/doc-debug").printDocToDebug;
 
+function guessLineEnding(text) {
+  const index = text.indexOf('\n');
+  if (index === -1) {
+    return "\n";
+  }
+  if (text.charAt(index - 1) === "\r") {
+    return "\r\n";
+  }
+  return "\n";
```

Comment:
This could be simplified as:

```js
const index = text.indexOf('\n');
if(index >= 0 && text.charAt(index - 1) === "\r") {
  return "\r\n";
}
return "\n";
```